### ISSUE TITLE
WCAG 3.0 Review: Feedback on question posed in Section 2 Normative requirements

### ISSUE THREAD

**Issue by maryjom** [w3c/silver#612]
### Section 2 Normative requirements

**Question:** Outcomes are normative. The working group is looking for feedback on whether the following should be normative or informative: guidelines, methods, critical errors, and outcome ratings.

**Answer:** 
- Guidelines - Informative. These are simply logical collections or ways of organizing the outcomes. They don’t provide the necessary details or exceptions as a requirement would or should. In addition, when I looked up definitions of "guideline", I find  the term to be used more in the realm of guidance, not rules or requirements. Example definitions I found:
    - A general rule, principle, or piece of advice
    - A plan or explanation to guide one in setting standards or determining a course of action.
    - A non-specific rule or principle that provides direction to action or behavior.
-  Critical errors – In most cases (all?)  these could be normative. However, one must be careful that critical errors aren’t technology-specific in a way that doesn’t allow for innovative alternate ways to conform.
- Methods – informative. This could potentially go either way (or be a mix of informative and normative, depending on the technology). There may be some instances where there is technically only one way to meet a requirement (e.g., requirements for marking up EPUB, or pdf). However, there could be many other cases where there are options to meet the guideline in alternate, creative ways (e.g., using plain HTML vs. using WAI-ARIA). In the past with WCAG 2.x, there were sometimes issues with the interpretation that specific techniques were required (at a time when ARIA was newly supported). We don’t want to have that situation occur in the future when there may be some novel ways introduced to implement accessibility to meet a particular requirement. Those should not be “disallowed” because one is required to follow a particular implementation method.
- Outcome ratings – Difficult to say on this. This is a two-edge sword. On one side having outcome ratings that are normative could help ensure different assessors are using the same rating scale. It allows for product comparisons. This would work for criteria that are very crisp on what the outcomes should be (e.g., color contrast with its table of acceptable contrasts per font size), how to test, and how to aggregate the occurrences of applicable content. However, where there is subjectivity involved, outcome ratings may be less easily defined. Making them normative may prove difficult due to interpretation differences or difficulty in definitive testing. It also may be difficult to use in reporting mechanisms such as an ACR because each criteria seems to have different ratings that have unique meaning.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).

**Comment by maryjom** [comment]
Since we haven't fully formed conformance and what will/won't be required in the latest draft, this can be closed.
